docs/Running-Hadoop-MapReduce-on-Tachyon.md

```diff
@@ -1,6 +1,8 @@
 ---
 layout: global
 title: Running Hadoop MapReduce on Tachyon
+nickname: Hadoop MapReduce
```

Comment:
Is this the title that shows up in the navbar? If so, then Apache projects should be prefixed with "Apache".